### ISSUE TITLE
inline documentation

### ISSUE THREAD

**Issue by aschweigert** [INN/Largo#393]
in 0.5 we should aim for the docs in largo to be more or less comprehensive; in 0.6 let's think more about inline docs and ways of displaying help messages in context to make content creators' lives easier.

**Comment by aschweigert** [comment]
For example, we *could* add contextual help to various admin page types (e.g. - http://ottopress.com/2011/new-in-wordpress-3-3-more-useful-help-screens/) the question is whether or not people actually know to look there.

We might also just want to think about using tooltips and various other hinting in the various metaboxes or custom UI elements we add so the docs don't get in the way but are easy to access when needed.

**Comment by meredithinn** [comment]
I think having a look at what exists in those help screens and determining what might be added that would be useful for Largo users might be a good first step.  Maybe including a mention of them in our docs somewhere?

**Comment by aschweigert** [comment]
Yeah, it wouldn't hurt to just outline what's already there (this may exist somewhere already, probably does, actually) but then we need to think about where we add OUR stuff, is it to these panels? or somewhere else? a combination of both?

**Comment by aschweigert** [comment]
@meredithinn a good read on this general idea: http://alistapart.com/article/training-the-cms

**Comment by meredithinn** [comment]
@aschweigert The current links from the Project Largo Help box on the Dashboard   http://projectlargo.org/questions/ and http://projectlargo.org/contact/ are both leading to Page Not Found

**Comment by aschweigert** [comment]
on the dashboard where?

**Comment by meredithinn** [comment]
In the Project Largo Help box at the bottom of the page.
![dashboard help](https://cloud.githubusercontent.com/assets/9658519/6028637/03ca943e-abb9-11e4-9c20-594e025dfcd4.png)

**Comment by aschweigert** [comment]
oh, i see

**Comment by meredithinn** [comment]
Created a brief outline of the existing inline help  -- interesting to identify the "blank" screens https://docs.google.com/a/investigativenewsnetwork.org/spreadsheets/d/1WY2g2-JswEHlK3aQxDfMx-qesVrmNAgCsxjt6MsXNec/edit#gid=0

**Comment by aschweigert** [comment]
so it seems like we should at least do:

custom taxonomies
- [ ] series
- [ ] post prominence
- [ ] post types

appearance menu
- [ ] theme options
- [ ] less/css 

and possibly plugins
- [ ] clean contact

and see if we can add the stuff we modify to the other menus
- [ ] categories (custom sidebars, post type icons)
- [ ] tags (same)
- [ ] posts/pages > add new (all the custom metaboxes, featured media, etc.)
- [ ] widgets?
- [ ] user profiles
